sdk/src/signing.rs

```diff
@@ -277,8 +277,16 @@ pub async fn aux_signing_data<'a>(
             Some(AccountPublicKeysMap::from_iter(public_keys.clone())),
             1u8,
         ),
-        Some(owner @ Address::Internal(_)) => {
-            return Err(Error::from(TxError::InvalidAccount(owner.encode())));
+        Some(owner @ Address::Internal(internal)) => {
+            match internal {
+                // TODO use sentinel?
```

Comment:
I don't think we need a signature on the transaction at all if the source is the MASP cause no VP would check it anyway